cmd/snap-recovery-chooser/main.go

```diff
@@ -65,14 +65,30 @@ var (
 
 // consoleConfWrapperUITool returns a hardcoded path to the console conf wrapper
```

Comment:
I feel like this comment doesn't reflect the truth anymore, we do not return a path?